adr/ADR-8.md

```diff
@@ -31,8 +31,8 @@ Initial feature list:
  * Encoders and Decoders that transforms both keys and values
  * A read-cache that builds an in-memory cache for fast reads
  * read-after-write safety unless read replicas used
- * Valid keys are `\A[-/_a-=zA-Z0-9]+\z` after encoding
- * Valid buckets are `^[a-zA-Z0-9_-]+$`
+ * Valid keys are `\A[\./-_a-zA-Z0-9]+\z` after encoding
+ * Valid buckets are `\A[a-zA-Z0-9_-]+\z`
```

Comment:
Same here.

Reply:
just avoids some newline related issues when validating the regex - intent stays the same but now the regex correctly captures the intent :)